src/audio/tdfb/tdfb.c

```diff
@@ -31,6 +31,12 @@
 #include <stddef.h>
 #include <stdint.h>
 
+/* The driver assigns running numbers for control index. The enum controls
+ * get indices 0 and 1. Make sure the on/off control is first in topology.
+ */
+#define ENUM_CTRL_ONOFF		0
+#define ENUM_CTRL_DIRECTION	1
```

Comment:
Should this be a real enum ?

Reply:
The control # index not an enum but the contents of on/off control and direction control are enums. I think I should rename these as e.g. CTRL_INDEX_ONOFF and CTLR_INDEX_DIRECTION.